src/server/detail/save_stages_controller.cc

```diff
@@ -129,6 +131,11 @@ size_t RdbSnapshot::GetSaveBuffersSize() {
   return saver_->GetTotalBuffersSize();
 }
 
+std::pair<size_t, size_t> RdbSnapshot::GetCurrentSnapshotProgress() const {
```

Comment:
How about returning a struct SnapshotStats, it will be more clear to reader and you wont need to add a comment what is the first and second param

Reply:
will do